### ISSUE TITLE
Add FAQ page to the website

### ISSUE THREAD

**Issue by LoriKarikari** [blitz-js/blitzjs.com#170]
We are getting a lot of repeated questions on Slack. It would be nice to have a FAQ page on the website to have an overview of questions that are commonly asked.

**Comment by liamjosephsilk** [comment]
I could pick this one up as well? Is there a curated list of common questions/answers?

**Comment by LoriKarikari** [comment]
There isn't one AFAIK. We should probably create one. I'm sure @flybayer already has some questions to add to that list.

**Comment by liamjosephsilk** [comment]
I'm still happy to pick this up?

**Comment by LoriKarikari** [comment]
Yeah, you can start working on it. We can provide you with the questions or add them afterwards I guess.

**Comment by flybayer** [comment]
I don't have a list of questions, but you can browse through the help channel in slack and pick ones that seem good to have 🤷

**Comment by pranikz** [comment]
Assign me I can pick this up

**Comment by flybayer** [comment]
@pranikz let's give @liamjosephsilk some time to finish it. Unless he wants to hand it off to you.

**Comment by liamjosephsilk** [comment]
Im happy to hand it off, still finishing up the tutorial @pranikz , @flybayer.

**Comment by pranikz** [comment]
okay sure no issue I can work on that.

**Comment by nonye94** [comment]
Hello, do you still need someone to complete this task.
